conv_model_fwd.cpp

```diff
@@ -287,6 +315,10 @@ int conv_benchmark(int argc, char** argv) {
   // JIT requested nested loop specs
 
   auto t0 = getTime();
+  auto input_pad_loop = ThreadedLoop<2>({
+      LoopSpecs{0, N, n_step, true},
+      LoopSpecs{0, Cb, c_step}},
+      "Ab");
```

Comment:
Possibly, if ever used sequentially (with "a" in the tuning string for the main conv loop, we would want t have lowercase "a" as well). Do we want to be safe here and put some checks here?

Reply:
I guess for now this is good. We may want to open future PRs that address such checks (i.e. code quality improvements)